presentation/src/main/java/com/android254/presentation/common/components/OrganizedBySection.kt

```diff
@@ -59,7 +62,7 @@ fun OrganizedBySection(
             modifier = Modifier.fillMaxWidth(),
             text = stringResource(id = R.string.organized_by),
             style = TextStyle(
-                color = ChaiBlue,
+                color = if (!isSystemInDarkTheme()) ChaiBlue else ChaiTeal,
```

Comment:
We shouldn't check if the system is in darkMode here, we can use `MaterialTheme.colorScheme` and pick the relevant colour needed that matches the design in both `Light` and `Dark` mode.
They can be found here : [Theme Colors ](https://github.com/droidconKE/droidconKE2023Android/blob/main/chai/src/main/java/com/droidconke/chai/Theme.kt)

> Please Note  
If you have looked at the design and confirmed there is no matching color you can create it and  :
- create a color variable using the example set for `textColorPrimary` found [here](https://github.com/droidconKE/droidconKE2023Android/blob/main/chai/src/main/java/com/droidconke/chai/colors/ChaiColors.kt)
- call your color variable from the composable using `MaterialTheme.chaiColorsPalette.{yourColorName}`